build/Dockerfile.nodeagent

```diff
@@ -1,8 +1,8 @@
-FROM golang:1.13.1
+FROM golang:1.15.14 as builder
```

Comment:
it doesn't seem `--from=0` uses the `builder` alias, and I don't think we should keep maintaining nodeagent anyway

Reply:
removed the builder
and for the nodeagent maintenance, I am not sure if it's in this ticket scope

Reply:
exactly, no need to make changes to it in this ticket scope, it's dead code

Reply:
@matancarmeli7 gentle reminder

Reply:
done